UPDATING.md

```diff
@@ -89,11 +89,11 @@ Smart sensors, an "early access" feature added in Airflow 2, are now deprecated
 
 See [Migrating to Deferrable Operators](https://airflow.apache.org/docs/apache-airflow/2.3.0/concepts/smart-sensors.html#migrating-to-deferrable-operators) for details on how to migrate.
 
-### Task log filenames are not rendered from database entry instead of config value
+### Task log templates are now read from the metadatabase instead of `airflow.cfg`
 
-Previously, filename of a task’s log is dynamically rendered from the ``[core] log_filename_template`` config value at runtime. This resulted in unfortunate characteristics like it is inpractical to modify the config value after an Airflow instance is running for a while, since all existing task logs have be saved under the previous format and cannot be found with the new config value.
+Previously, a task’s log is dynamically rendered from the `[core] log_filename_template` and `[core] task_log_prefix_template` config values at runtime. This resulted in unfortunate characteristics, e.g. it is impractical to modify the config value after an Airflow instance is running for a while, since all existing task logs have be saved under the previous format and cannot be found with the new config value.
```

Comment:
```suggestion
Previously, a task’s log is dynamically rendered from the `[core] log_filename_template` and `[core] task_log_prefix_template` config values at runtime. This resulted in unfortunate characteristics, e.g. it is impractical to modify the config value after an Airflow instance is running for a while, since all existing task logs have been saved under the previous format and cannot be found with the new config value.
```